### DISCUSSION TITLE
Check my new repository for any related challenges/katas in CodeWars

### DISCUSSION THREAD

**Discussion by vvxxrrvvx** [community/community#61544] [GitHub Education]
### Hi everyone,

https://github.com/vvxxrrvvx/codewars-kata

**Comment by JosueEstrada** [comment]
Hi! I'm going to borrow your idea. I've been on Codewars for a long time solving Python and JS katas, and it never occurred to me to collect them to show them. It's good for the resume.

Something to improve could be to add a webpage and show the code with clipboard button, formated with markdown block code (?) and the link to the kata in cards, with filters for each language, etc."

**Reply by vvxxrrvvx** [reply]
Thank you for the feedback. I'll give the links to the related katas in the repo.
